### PR TITLE
fix(DB/smart_scripts):  Quest Redemption Text and Animation Missing Fix

### LINKED ISSUE #10649: Quest Redemption Text and Animation Missing
### Current Behaviour

The quest is completed 
Only the texts and the running is missing

### Expected Blizzlike Behaviour

The shaman should be revived. 
After that he should look at you.
Say a text and then run away

### Source

Video is in German unfortunately no other found.

https://youtu.be/t7XbU2Tvc7Y?list=PL5pCQOnqlcrc0UhrmyJ6iyMLhdWE5hYB5&t=809

The texts are missing in creature_text 
But are present in broadcast_text 

Id 14110 Text Type is 12
Id 14109 Text Type is 16

NPC ID 17542

### Steps to reproduce the problem

Give yourself the quest Redemption

.quest add 9600
.go c 63139
Use the item from the quest

### Extra Notes

_No response_

### AC rev. hash/commit

('AzerothCore rev. 87f76587d808 2022-02-07 06:39:50 -0300 (master branch) (Win64, RelWithDebInfo, Static)', '87f76587d808', 'ACDB 335.6-dev', 6);

### Operating system

Windows 10

### Custom changes or Modules

None

### PR DESCRIPTION
**Pull request by Selyria**
Fix Say and run event

DESCRIPTION
NPC should look at you after rebirth and say a text and run away and despawn
Example: Young Furbolg Shaman  (id: 17542)

Closes #10649

Have tested it myself and am satisfied

Sorry if I made a mistake Last fix was 5 years ago

### CODE REVIEW

**Comment by Lynethris** [comment]
Tested and work. Thanks for your PR :)

**Comment by Lynethris** [comment]
Sorry, multiple edit. I was thinking that the corpse didn't respawn. But everything is good. Thanks for you PR :)

File: data/sql/updates/pending_db_world/rev_1649949817142190900.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+INSERT INTO `waypoints` (`entry`,`pointid`,`position_x`,`position_y`,`position_z`,`orientation`,`delay`,`point_comment`) VALUES
+(17542,1,-2444.689,-12207.18,32.8721,0,0,"Young Furbolg Shaman"),
+(17542,2,-2443.185,-12220.92,31.8721,0,0,"Young Furbolg Shaman"),
+(17542,3,-2435.321,-12240.7,32.0348,0,0,"Young Furbolg Shaman"),
+(17542,4,-2423.951,-12278.52,27.72987,0,0,"Young Furbolg Shaman");
+
+DELETE FROM `creature_text` WHERE `CreatureID` = 17542;
+INSERT INTO `creature_text` (`CreatureID`,`GroupID`,`ID`,`Text`,`Type`,`Language`,`Probability`,`Emote`,`Duration`,`Sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(17542,0,0,"%s stares at you in sheer wonderment.",16,0,100,0,0,0,14109,0,"Young Furbolg Shaman"),
+(17542,1,0,"I run now!",12,0,100,0,0,0,14110,0,"Young Furbolg Shaman");
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1649949817142190900.sql:29]:
```suggestion
(17542,1,0,"I run now!",12,0,100,0,0,0,14110,0,"Young Furbolg Shaman");

```

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1649949817142190900.sql:29]: add newline at the end of the file.

Comment (Selyria) [reply data/sql/updates/pending_db_world/rev_1649949817142190900.sql]: okay thanks

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**